cypress-tests/cypress/support/utils/platform/multiEnv.js

```diff
@@ -2,6 +2,28 @@ import { multiEnvSelector, commonEeSelectors } from "Selectors/eeCommon";
 import { commonSelectors, commonWidgetSelector } from "Selectors/common";
 import { appVersionSelectors } from "Selectors/exportImport";
 import { appVersionText } from "Texts/exportImport";
+import { multiEnvText } from "Texts/eeCommon";
+
+const environments = {
+    development: "development",
+    staging: "staging",
+    production: "production",
+};
+
+const widgetPositions = {
```

Comment:
I can see the same constants in spec as well. Please review it yourself before assigning for review.